ref/README.md

```diff
@@ -79,6 +84,27 @@ On many systems, the above will suffice. However, some systems will require you
 find dependencies, particularly if software depencencies are not installed in standard locations.
 See below for more information.
 
+### Building with GPTL profiling support
+
+If GPTL is installed and loaded onto system paths, this kernel may be built with support for
+GPTL profiling. There are two options.
+
+The first option enables GPTL and allows you to	insert your own	custom GPTL timers into the code
```

Comment:
I would suggest writing about autoprofiling first, because that is the most common use.  The "insert your own custom GLTP timers" is rarer.